packages/shared/src/interface/external/surescripts/payment-code.ts

```diff
@@ -22,3 +26,56 @@ export const PaymentCodeName: Record<PaymentCode, PaymentCodeName> = {
   "07": "Indian Nations",
   "99": "Other",
 };
+
+export function getSourceOfPaymentCode(paymentCode: string): SourceOfPaymentCode | undefined {
+  return PaymentCodeToSourceOfPaymentCode[paymentCode as PaymentCode];
+}
+
+export function getSourceOfPaymentName(sourceOfPaymentCode: string): string | undefined {
+  return SourceOfPaymentName[sourceOfPaymentCode as SourceOfPaymentCode];
+}
+
+// https://www.nahdo.org/sopt
+const SourceOfPaymentCodes = [
+  "1",
+  "2",
+  "3",
+  "312",
+  "33",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "89",
+  "95",
+  "99",
+] as const;
+export type SourceOfPaymentCode = (typeof SourceOfPaymentCodes)[number];
+
+export const SourceOfPaymentName: Record<SourceOfPaymentCode, string> = {
+  "1": "Medicare",
+  "2": "Medicaid",
+  "3": "Government",
```

Comment:
On the doc I see "3 or 4" = "government or corrections". Is there more detail that leads to saying that 3 is gvmt and 4 is corrections?

Reply:
If you see the full table I exported here for the code system: https://www.notion.so/metriport/Source-of-Payment-Typology-21db51bb90408076a4c1e8be2f78a264 

"3" is the code for all government payers specifically excluding DOC, and "4" is specifically for corrections.